TypeCobol/Compiler/CupPreprocessor/CompilerDirectiveBuilder.cs

```diff
@@ -90,7 +90,7 @@ public virtual void EnterCopyCompilerStatement(TypeCobol.Compiler.Scanner.Token
             var copy = (CopyDirective)CompilerDirective;
             bool isCopy = copy.COPYToken.TokenType == TokenType.COPY;
             copy.TextName = GetName(qualifiedTextName.TextName);
-            copy.TextNameSymbol = qualifiedTextName.TextName;
+            copy.TextNameSymbol = qualifiedTextName.TextName;          
```

Comment:
Redundant spaces at the end of the line.

Reply:
fixed : new commit
https://github.com/TypeCobolTeam/TypeCobol/tree/774bbecec9a370b3d43ebfac4c3d9ab38cc7e072